ff --git a/libcaf_io/caf/io/network/default_multiplexer.hpp b/libcaf_io/caf/io/network/default_multiplexer.hpp
index c85291b9..777426a0 100644
--- a/libcaf_io/caf/io/network/default_multiplexer.hpp
+++ b/libcaf_io/caf/io/network/default_multiplexer.hpp
@@ -62,7 +62,7 @@ struct epoll_event;
 #if !defined(CAF_LINUX) || defined(CAF_POLL_IMPL)
 #define CAF_POLL_MULTIPLEXER
 #else
-#define CAF_EPOLL_MULTIPLEXER
+#define CAF_POLL_MULTIPLEXER
 #endif
 
namespace caf {
